packages/governance/test/unitTests/test-buildParamManager.js

```diff
@@ -99,7 +99,7 @@ test('Amount', async t => {
   await t.throwsAsync(
     () => paramManager.updateParams({ Shimmer: 'fear,loathing' }),
     {
-      message: 'Expected an Amount for Shimmer, got "fear,loathing"',
+      message: 'Expected an Amount for "Shimmer", got ""fear,loathing""',
```

Comment:
why are the quotes doubled? (or quadrupled as it were)

Reply:
The amount is usually something other than a string, so the error message includes quotes to set them off for distinctness. Whoever is substituting the string from the details template thinks they should have quotes. I agree with the latter, and don't mind double-doublequotes before and after.

Reply:
That's fine. But what I usually do for such cases at the end of a diagnostic string is to set it off with a colon rather than quotes. I do not have any strong preference though.
```js
      assert(a.brand, X`Expected an Amount for ${q(name)}, got: ${a}`);
```

Oh, one more thing. For this ava error message testing, I have come to prefer regexps rather than literal strings, so that can be more tolerant of variations extraneous to the purpose of the test. For *example*, depending on the purpose of the test, this might be

```suggestion
      message: /Expected an Amount for .*, got .*/,
```

Without knowing more about the purpose of this test, I am *not* suggesting this. Just illustrating for when it is helpful.